tools/test-case-generators/generate-test-case

```diff
@@ -41,9 +41,10 @@ def run_osbuild(manifest, store, output):
                     "--store", store,
                     "--output-directory", output,
                     "--json", "-"],
-                   check=True,
-                   encoding="utf-8",
-                   input=json.dumps(manifest))
+                    capture_output=True,
```

Comment:
we don't use the captured output, do we? I would prefer `stdout=subprocess.DEVNULL` then.

Reply:
Sure, I'll change it :)